src/olympia/amo/tests/__init__.py

```diff
@@ -356,6 +356,10 @@ def days_ago(days):
 
 
 def start_es_mocks():
+    # Before starting to mock, stop them first. That way we ensure we're not
+    # trying to mock over an existing mock, which would be problematic since
+    # we use spec=True.
+    stop_es_mocks()
```

Comment:
Nit but... this seems a bit wasteful when we know the es mocks won't have started when it's called from `TestCase`, and wasteful a second time when we're stopping the mocks and starting them again in the autouse fixture.  Is there a way of checking that we've initialized the mocks once already?  If we can't think of anything I guess it'd be a future enhancement kind of thing.

Reply:
I tried... there is no public API to check that a patcher has been started :(